types/office-js-preview/index.d.ts

```diff
@@ -17991,12 +17991,11 @@ declare namespace Office {
          *
          * **Important**:
          *
-         * - Legacy Exchange tokens are deprecated. Legacy Exchange {@link https://learn.microsoft.com/office/dev/add-ins/outlook/authentication#exchange-user-identity-token | user identity} and
-         * {@link https://learn.microsoft.com/office/dev/add-ins/outlook/authentication#callback-tokens | callback} tokens have been turned off by default for most Exchange Online tenants.
-         * Administrators can reenable legacy tokens for tenants and add-ins until June 2025. In October 2025, legacy tokens will be completely turned off for all tenants.
-         * This is part of {@link https://blogs.microsoft.com/on-the-issues/2023/11/02/secure-future-initiative-sfi-cybersecurity-cyberattacks/ | Microsoft's Secure Future Initiative},
-         * which gives organizations the tools needed to respond to the current threat landscape. Exchange user identity tokens will still work for Exchange on-premises.
-         * Nested app authentication (NAA) is the recommended approach for tokens going forward. For more information, see the {@link https://aka.ms/naafaq | FAQ page}.
+         * - Legacy Exchange Online {@link https://learn.microsoft.com/office/dev/add-ins/outlook/authentication#exchange-user-identity-token | user identity tokens} and
+         * {@link https://learn.microsoft.com/office/dev/add-ins/outlook/authentication#callback-tokens | callback tokens} are no longer supported and turned off across all Microsoft 365 tenants.
```

Comment:
here and other instances...

```suggestion
         * {@link https://learn.microsoft.com/office/dev/add-ins/outlook/authentication#callback-tokens | callback tokens} are no longer supported and are turned off across all Microsoft 365 tenants.
```

Reply:
Updated.